stable/appmesh-gateway/README.md

```diff
@@ -0,0 +1,100 @@
+# App Mesh Gateway
+
+App Mesh Gateway Helm chart for Kubernetes
+
+## Prerequisites
+
+* App Mesh CRDs
+* App Mesh Manager >= 1.0.0
+
+## Installing the Chart
+
+Add the EKS repository to Helm:
+
+```sh
+helm repo add eks https://aws.github.io/eks-charts
+```
+
+Create a namespace with injection enabled:
+
+```sh
+kubectl create ns appmesh-ingress
+kubectl label namespace appmesh-ingress appmesh.k8s.aws/sidecarInjectorWebhook=enabled
```

Comment:
Would be good to make `mesh` configurable since a cluster may have multiple meshes and each mesh having it's own set of virtual gateways